samples/model-builder/predict_tabular_classification_sample.py

```diff
@@ -22,9 +22,17 @@
 def predict_tabular_classification_sample(
     project: str, location: str, endpoint: str, instances: List[Dict],
 ):
+    '''
+    Args
+        project: Your project ID or project number.
+        location: Region where Endpoint is located. For example, 'us-central1'.
+        endpoint: A fully qualified endpoint name or endpoint ID. Example: "projects/123/locations/us-central1/endpoints/456" or
```

Comment:
Docstring also need to reflect the change from `endpoint` to `endpoint_name`

Reply:
fixed

Reply:
fixed